src/fklearn/training/classification.py

```diff
@@ -313,12 +328,9 @@ def catboost_classification_learner(df: pd.DataFrame,
 
     def p(new_df: pd.DataFrame, apply_shap: bool = False) -> pd.DataFrame:
 
-        dtest = Pool(new_df[features].values, feature_names=list(map(str, features)),
-                     cat_features=cat_features)
-
-        pred = cbr.predict_proba(dtest)[:, 1]
+        pred = cbr.predict_proba(new_df[features])[:, 1]
```

Comment:
this should be inside the `else` in L325/R337. In the `if` case (MultiClass) the prediction is calculated twice, with overwriting.